src/concrete_intersection.jl

```diff
@@ -550,3 +550,31 @@ end
 function intersection(S::AbstractSingleton{N}, U::Universe{N}) where {N<:Real}
     return S
 end
+
+"""
+    intersection(P::AbstractPolyhedron{N}, rm::ResetMap{N}) where {N<:Real}
+
+Return the intersection of a polyhedron and a polytopic reset map.
+
+### Input
+
+- `P`  -- polyhedron
+- `rm` -- polytopic reset map
+
+### Output
+
+An `HPolytope`.
+
+### Notes
+
+We assume that `rm` is polytopic, i.e., is bounded and has a `constraints_list`
+method defined.
+"""
+function intersection(P::AbstractPolyhedron{N}, rm::ResetMap{N}) where {N<:Real}
+    return HPolytope([constraints_list(P); constraints_list(rm)])
```

Comment:
Have you considered removing redundant constraints? This is done by default for polytope intersections [here](https://github.com/JuliaReach/LazySets.jl/blob/master/src/concrete_intersection.jl#L340) and i would suggest to use it here as well.

Reply:
Perhaps we can just invoke that function but doing `HPolytope(constraints_list(rm))`?

Reply:
Yes, I considered it, but decided against it.
I changed the code (it was actually unsound) and now it uses the method you linked to.

Reply:
Great, looks good to me. The argument behind removing redundant constraints by deafult is that is simplifies a lot of cases where the intersection is actually empty. And, well if the intersection is not empty you get a simpler polyhedron. But we can add a flag to turn this on or off, currently there is no such flag.